### ISSUE TITLE
Audio for the World Is Not Enough Not Working

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#525]
Originally reported on Google Code with ID 526

```

 - Operating System: Mac OS X Mountain Lion
 - Machine type: 64 Bit
 - Mupen64Plus version: 0.3 With GUI
 - Plugins used: All Default Plugins

Hi! Just wanted to start by saying Mupen64Plus is my favorite N64 emulator :)  Whenever
I try to play The World Is Not Enough (James Bond Game) I can get the video to work
by switching to the Glyde video plugin, but I cannot get the audio to work.  Is there
a way to fix this so I can experience audio with the game?  If so, please help me out.
 Thanks (:
```

Reported by `MoneyMan56@me.com` on 2012-12-05 23:26:22

**Comment by richard42** [comment]
```
 I am not sure about this game but did you try to use mupen64plus-rsp-z64hlevideo? I
may have mistyped the correct name but is is a special compilation of mupen64plus-rsp-z64
but compatible with rice, arachnoid and glide64. It still does the audio emulation
as low level emulation and therefore can support slightly more ucodes than the high
level emulation of mupen64plus-rsp-hle.  But it is slower and maybe has other bugs
```

Reported by `sven@narfation.org` on 2012-12-06 06:56:28

**Comment by richard42** [comment]
```
Just checked the name again and it is called "mupen64plus-rsp-z64-hlevideo"

Can be compiled from https://bitbucket.org/wahrhaft/mupen64plus-rsp-z64 with the option
HLEVIDEO=1 passed to the make.
```

Reported by `sven@narfation.org` on 2012-12-06 10:08:17

**Comment by richard42** [comment]
```
Thanks for the help, but I am not sure how to convert that file to a readable .dylib
.  How do you make that file conversion?
```

Reported by `MoneyMan56@me.com` on 2012-12-06 21:24:27

**Comment by richard42** [comment]
```
You have to compile it as you would normally do it on MacOS ( http://code.google.com/p/mupen64plus/wiki/MacOSXInstructions
). Maybe the author (Auria) of wxmupen64plus can help you
```

Reported by `sven@narfation.org` on 2012-12-06 22:10:33

**Comment by richard42** [comment]
```
Holy smokes, that is a lot of information.  I'll try my best and let you how it turns
out, Thx =)
```

Reported by `MoneyMan56@me.com` on 2012-12-06 22:12:40

**Comment by richard42** [comment]
```
I'm not good with coding and such, is there any program or easier way to compile the
file into the dylib?
```

Reported by `MoneyMan56@me.com` on 2012-12-09 20:35:20

**Comment by richard42** [comment]
```
I have no idea what is necessary on MacOSX to get it compiled. It is probably better
ask someone to compile it for you (Auria? Richard42?)
```

Reported by `sven@narfation.org` on 2012-12-10 12:28:46

**Comment by richard42** [comment]
Reported by `sven@narfation.org` on 2013-12-18 12:09:53
- **Status changed**: `Duplicate`
- **Merged into**: #99
